app/assets/javascripts/discourse/app/components/user-stream.gjs

```diff
@@ -0,0 +1,233 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { fn, hash } from "@ember/helper";
+import { action } from "@ember/object";
+import { getOwner } from "@ember/owner";
+import { later } from "@ember/runloop";
+import { service } from "@ember/service";
+import { modifier } from "ember-modifier";
+import $ from "jquery";
+import DButton from "discourse/components/d-button";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import PostActionDescription from "discourse/components/post-action-description";
+import PostList from "discourse/components/post-list";
+import avatar from "discourse/helpers/avatar";
+import concatClass from "discourse/helpers/concat-class";
+import dIcon from "discourse/helpers/d-icon";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import ClickTrack from "discourse/lib/click-track";
+import DiscourseURL from "discourse/lib/url";
+import { NEW_TOPIC_KEY } from "discourse/models/composer";
+import Draft from "discourse/models/draft";
+import Post from "discourse/models/post";
+import { i18n } from "discourse-i18n";
+
+export default class UserStreamComponent extends Component {
+  @service dialog;
+  @service composer;
+  @service appEvents;
+  @service currentUser;
+  @tracked lastDecoratedElement;
+  @tracked filterClassName = "";
+
+  eventListeners = modifier((element) => {
+    $(element).on("click.details-disabled", "details.disabled", () => false);
+    $(element).on("click.discourse-redirect", ".excerpt a", (e) => {
+      return ClickTrack.trackClick(e, getOwner(this));
+    });
+    later(() => {
+      this.updateLastDecoratedElement();
+      this.appEvents.trigger("decorate-non-stream-cooked-element", element);
+    });
+
+    return () => {
+      $(element).off("click.details-disabled", "details.disabled");
+      // Unbind link tracking
+      $(element).off("click.discourse-redirect", ".excerpt a");
+    };
+  });
+
+  constructor() {
+    super(...arguments);
+    this.setFilterClassName();
+  }
+
+  setFilterClassName() {
+    const filter = this.args.stream?.filter;
+    if (filter) {
+      this.filterClassName = `filter-${filter.toString().replace(",", "-")}`;
+    }
+  }
```

Comment:
```suggestion
  get filterClassName(){
    const filter = this.args.stream?.filter;
    if (filter) {
      return `filter-${filter.toString().replace(",", "-")}`;
    }
  }
```
This would work instead, right?

(and drop the `filterClassName` tracked property)